boost-maven-plugin/src/main/java/io/openliberty/boost/docker/DockerPushMojo.java

```diff
@@ -49,8 +49,11 @@ private void pushDockerImage(DockerClient dockerClient) throws MojoExecutionExce
             Scanner in = new Scanner(System.in);
             String newRepository = in.next();
             if (newRepository == null) {
-                throw new NullPointerException("Repository name cannot be null");
+                throw new MojoExecutionException("The repository name cannot be null");
             }
+            if (!isRepositoryValid(newRepository)){
+                throw new MojoExecutionException("The repository name is invalid.");         
```

Comment:
should say "not valid" instead of "invalid"

Reply:
done.